Tests/MapboxMapsTests/Foundation/Extensions/UIWindow+ParentSceneTests.swift

```diff
@@ -0,0 +1,20 @@
+import XCTest
+import CarPlay
+@testable import MapboxMaps
+
+final class UIWindowParentSizeTests: XCTestCase {
+
+    @available(iOS 13.0, *)
+    func testCarPlayWindowReturnsCorrectParentScene() {
+        let window = CPWindow()
+
+        XCTAssertEqual(window.parentScene, window.templateApplicationScene)
+    }
+
+    @available(iOS 13.0, *)
+    func testUIWindowReturnsCorrectParentScene() {
+        let window = UIWindow()
+
+        XCTAssertEqual(window.parentScene, window.windowScene)
```

Comment:
Same question here about the actual values.

Reply:
`windowScene` seems to be a special property as it gets populated even when creating a window directly. So in this case bot `parentScene` and `windowScene` won't be nil.